apps/mobile/ios/leatherwalletmobile/Info.plist

```diff
@@ -2,8 +2,10 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
   <dict>
+    <key>ITSAppUsesNonExemptEncryption</key>
+    <false />
```

Comment:
Thanks @edgarkhanzadian 👍  Maybe we can take this change into a smaller PR so we can get it merged sooner? 

We didn't do another Testflight release since last week and there are some bug fixes we need to get out pre-Leatherhood.

Reply:
I picked and merged this in https://github.com/leather-io/mono/pull/672